src/core/FxMixer.cpp

```diff
@@ -312,6 +313,22 @@ void FxMixer::deleteChannel( int index )
 				inst->effectChannelModel()->setValue(val-1);
 			}
 		}
+		else if( t->type() == Track::SampleTrack )
+		{
+			SampleTrack* strk = dynamic_cast<SampleTrack *>( t );
```

Comment:
The spaces in the brackets here are really inconsitent in the block, it's not a violation of the coding conventions, but it just looks weird.